h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -1087,6 +1089,18 @@ public void setZValues(double [] zValues, double dispersion, boolean dispersionE
     double [][] _vcov;
     private double _dispersion;
     private boolean _dispersionEstimated;
+    public ArrayList<ScoreKeeper> _scored_train;
+    public ArrayList<ScoreKeeper> _scored_valid;
+    public ArrayList<ScoreKeeper> _scored_xval;
+    public TwoDimTable _scoring_history_early_stop;
+    public ScoreKeeper[] scoreKeepers() {
```

Comment:
I use ArrayList for scored_train/valid/xval.  GLM iteration is fast and hence if we are to copy and extend the array every time, it can be slow???  Not sure.  I understand ArrayList has more overhead then arrays but could be worth it.  Please let me know guys.